### ISSUE TITLE
Support Activation/deactivation in offline mode

### ISSUE THREAD

**Issue by pstibrany** [Tasktop/Foglyn#37]
While in offline mode, Foglyn should remember when task was activated and deactivated and submit these to FogBugz server when it is available again.

**Comment by elklaaso** [comment]
I agree, please can somebody do this? I always have to check if activation went through...with a short internet drop out my time doesnt get tracked correctly and i used this for invoicing!
I do not mind to pay for this feature
